classes/components/forms/context/PKPDoiSetupSettingsForm.inc.php

```diff
@@ -87,12 +110,23 @@ public function __construct(string $action, array $locales, Context $context)
                     ]
                 ],
                 'value' => $context->getData(Context::SETTING_DOI_CREATION_TIME) ? $context->getData(Context::SETTING_DOI_CREATION_TIME) : Repo::doi()::CREATION_TIME_COPYEDIT,
-                'showWhen' => Context::SETTING_ENABLE_DOIS
-
             ]))
-            ->addField(new FieldOptions(Context::SETTING_CUSTOM_DOI_SUFFIX_TYPE, [
+            ->addField(new FieldOptions(Context::SETTING_DOI_USE_DEFAULT_SUFFIX, [
                 'label' => __('doi.manager.settings.doiSuffix'),
+                'description' => __('doi.manager.settings.doiSuffix.default.description'),
+                'groupId' => self::DOI_SETTINGS_GROUP,
+                'type' => 'radio',
+                'options' => [
+                    ['value' => true, 'label' => __('common.yes')],
+                    ['value' => false, 'label' => __('common.no')]
+                ],
+                'value' => (bool) $context->getData(Context::SETTING_DOI_USE_DEFAULT_SUFFIX),
+            ]))
```

Comment:
I think at this point we want this setting to go away, right? The "default suffix pattern" is no longer a thing, and any previous setting of default should be migrated to the "user-defined patterns" setting.